README.md

```diff
@@ -1,2 +1,226 @@
 # Korn
 Korn: an opinionated Konflux Operator Release application
+
+# Building the binary
+
+To build the binary, just run the `make build` target and copy the binary in the `output/` directory to somewhere in your path.
+
+# Motivation
+
+Releasing an operator with Konflux generally translates into a bespoke process involving a set of unique checks and validations that are intertwined with Konflux's own domain constructs (snapshots, releases, release plan and release plan admissions). As such, Konflux only enforces its validations on what's defined by the Enterprise Contract Plan and the RPA's rules, but it is left to the operator's release team to ensure that the release artifacts are consistent, for instance that the image pullspec in the snapshots match the value in the bundle's reference images. This leaves up to the release engineer to define what validations or processes to set in place in order to guarantee the success of a release. In the context of releasing an Operator, the following validations apply in principle:
+
+* The Snapshot object needs to have been created from a push event.
+* The Snapshot object needs to be successful.
+* The component images referenced in the snapshot exist.
+* The related images in the CSV contained in the bundle container image referenced in the snapshot need to match the image references in the snapshot's spec. Failing this validation translates into image specs being pushed to production by the Konflux's release pipeline and the deployment of the operator failing to pull the correct ones due to digest mismatch.
+* All version labels found in each of the compoment's container image match.
+* Populate the `releaseNotes` in the Release manifest.
+
+Any snapshot that fails any validation cannot be used for a release.
+
+Korn's purpose is to simplify the process of releasing an operator in Konflux by implementing these validations.
+
+# Using korn to release an Operator with Konflux
```

Comment:
* What about adding a prerequisite section?
* nit: What about adding a short developer guide? (how to build/test)